docs/DeveloperGuide.adoc

```diff
@@ -117,9 +118,15 @@ image::EnqueueSequenceDiagram.png[]
 .Interactions inside the Logic Component for the `next 1` Command
```

Comment:
Do you think that it might be better to have a gap between the logic and model class to make it more readable? You can also consider splitting up the diagram as the fonts are too small at the moment.